locales/en/cms.ftl

```diff
@@ -9,7 +9,7 @@ l10n-1-EmailFirstPage-headline-f11ac15c = Make Space for What Matters
 # Primary Button Text for Email First Page
 l10n-1-EmailFirstPage-primaryButtonText-29ff216d = Create My List
 # Description for New Device Login Email
-l10n-1-NewDeviceLoginEmail-description-cdc80f7f = Your Mozilla account was used to sign in to 123Done
+l10n-1-NewDeviceLoginEmail-description-e11f8ec5 = Your Mozilla account was used to sign in to 123Done
```

Comment:
Filed https://mozilla-hub.atlassian.net/browse/FXA-12474 to address the new ftl id being generated for same string.